Brooklyn/Contexts/Preferences/fr.lproj/PreferencesWindowController.strings

```diff
@@ -0,0 +1,72 @@
+
+/* Class = "NSTextFieldCell"; title = "Want to keep updated?"; ObjectID = "3ad-ri-H5Y"; */
+"3ad-ri-H5Y.title" = "Voulez-vous rester à jour?";
```

Comment:
Not a huge fan of Localizable xibs as they will spread our translations across multiple files. Can we have them all in `Localizable.strings`?

Reply:
I've removed the xib location files and converted all references to using Localizable.strings.

Reply:
Yeah I noticed, the only thing "missing" is this:
https://github.com/pedrommcarrasco/Brooklyn/pull/60#discussion_r332912689